eclair-core/src/main/scala/fr/acinq/eclair/channel/fund/InteractiveTxBuilder.scala

```diff
@@ -1092,10 +1093,11 @@ object InteractiveTxSigningSession {
   case class WaitingForSigs(fundingParams: InteractiveTxParams,
                             fundingTxIndex: Long,
                             fundingTx: PartiallySignedSharedTransaction,
+                            commitInput: InputInfo,
```

Comment:
This is wip. I think we should add `CommitmentFormat` to `InteractiveTxParams` at some point? Which will allow us to remove this `commitInput`.